lib/tasks/b2b.rake

```diff
@@ -0,0 +1,19 @@
+namespace :b2b do
+  desc 'Create corporate article type'
+  task create_type: :environment do
+    Rails.logger = ActiveRecord::Base.logger = Logger.new(STDOUT)
```

Comment:
If you're redirecting the rails logger to `STDOUT` you may as well just `puts` these messages.